<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pref_window</class>
 <widget class="QWidget" name="pref_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Gryptonite.qrc">
    <normaloff>:/icons/main.png</normaloff>:/icons/main.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="autoload">
         <property name="toolTip">
          <string>Automatically open the last file you used?</string>
         </property>
         <property name="whatsThis">
          <string>Check this box if you want this program to automatically open the last file you used.</string>
         </property>
         <property name="text">
          <string>Open last file on startup</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="autoLaunch">
         <property name="toolTip">
          <string>If there is a URL should it be opened in a web browser?</string>
         </property>
         <property name="whatsThis">
          <string>Whenever the small popout window opens, it can automatically launch a URL in a web browser if you check this box.</string>
         </property>
         <property name="text">
          <string>Auto-launch URLs in web browser</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="minimize_on_close">
         <property name="toolTip">
          <string>Should the application hide inside the tray icon, or just close?</string>
         </property>
         <property name="whatsThis">
          <string>When you click the &quot;X&quot; in the corner to close the program, having this box checked will make it so the program doesn't close, but instead hides inside the tray icon for convenient access.</string>
         </property>
         <property name="text">
          <string>Close button minimizes to tray</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="timeFrmt">
         <property name="toolTip">
          <string>Date format preference</string>
         </property>
         <property name="whatsThis">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;When displaying times in the main window tree, if this box is checked they wil appear in 24-hour time format.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>24-hour time format</string>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="toolTip">
            <string>How much time after hiding to tray should the program wait before automatically locking your secrets?</string>
           </property>
           <property name="text">
            <string>Lockout delay timer:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeoutDisplay">
           <property name="toolTip">
            <string>How much time after hiding to tray should the program wait before automatically locking your secrets?</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="delaySlider">
         <property name="toolTip">
          <string>How much time after hiding to tray should the program wait before automatically locking your secrets?</string>
         </property>
         <property name="whatsThis">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Use the slider to adjust the delay between the time when you minimize this application to the system tray and when it locks your secrets from access.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;You can disable this feature by moving the slider all the way left.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="toolTip">
            <string>How long should secret content remain in the system clipboard?</string>
           </property>
           <property name="text">
            <string>Clipboard reset timer:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="clip_timeoutDisplay">
           <property name="toolTip">
            <string>How long should secret content remain in the system clipboard?</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="clipboard_slider">
         <property name="toolTip">
          <string>How long should secret content remain in the system clipboard?</string>
         </property>
         <property name="whatsThis">
          <string>When you send an item to your clipboard, this application starts a timer that will clear the clipboard contents after the specified timeout.

You can disable this feature by moving the slider all the way left.</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="favorites">
      <attribute name="title">
       <string>Favorites</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Arrange them however you like:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QListView" name="listView">
           <property name="toolTip">
            <string>Your favorites will always be presented to you in the order shown here</string>
           </property>
           <property name="whatsThis">
            <string>This view shows the order in which your favorites will appear in the system tray menu (opened by right-clicking the icon).

Rearrange them here by clicking the up and down arrows.  Remove an item from favorites by clicking the red &quot;X&quot; button.</string>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::IgnoreAction</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="toolTip">
              <string>Move the item up in the list</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Gryptonite.qrc">
               <normaloff>:/icons/uparrow.png</normaloff>:/icons/uparrow.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="toolTip">
              <string>Move the item down in the list</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Gryptonite.qrc">
               <normaloff>:/icons/downarrow.png</normaloff>:/icons/downarrow.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_9">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Remove a favorite</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Gryptonite.qrc">
               <normaloff>:/icons/redX.png</normaloff>:/icons/redX.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Accept changes</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Gryptonite.qrc">
         <normaloff>:/icons/checkmark.png</normaloff>:/icons/checkmark.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="toolTip">
        <string>Cancel changes</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Gryptonite.qrc">
         <normaloff>:/icons/exclamation.png</normaloff>:/icons/exclamation.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Gryptonite.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>pref_window</receiver>
   <slot>accept_changes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>pref_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoload</sender>
   <signal>stateChanged(int)</signal>
   <receiver>pref_window</receiver>
   <slot>autoload(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoLaunch</sender>
   <signal>stateChanged(int)</signal>
   <receiver>pref_window</receiver>
   <slot>autolaunch(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeFrmt</sender>
   <signal>stateChanged(int)</signal>
   <receiver>pref_window</receiver>
   <slot>dateformat(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delaySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pref_window</receiver>
   <slot>slider_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>pref_window</receiver>
   <slot>delete_favorite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minimize_on_close</sender>
   <signal>stateChanged(int)</signal>
   <receiver>pref_window</receiver>
   <slot>minimize_close(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clipboard_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pref_window</receiver>
   <slot>clipboard_slider_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>pref_window</receiver>
   <slot>move_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>pref_window</receiver>
   <slot>move_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>accept_changes()</slot>
  <slot>autoload(int)</slot>
  <slot>autolaunch(int)</slot>
  <slot>dateformat(int)</slot>
  <slot>slider_changed(int)</slot>
  <slot>delete_favorite()</slot>
  <slot>minimize_close(int)</slot>
  <slot>clipboard_slider_changed(int)</slot>
  <slot>move_up()</slot>
  <slot>move_down()</slot>
 </slots>
</ui>
